connectors/opa/opa_connector.go

```diff
@@ -239,20 +228,36 @@ func buildNewEnfrocementAction(transformAction interface{}) (*pb.EnforcementActi
 			log.Println("Warning: unknown format of used policy information. Skipping policy", action)
 		}
 
-		if result, ok := action["result"].(string); ok {
+		if result, ok := action["action_name"].(string); ok {
 			switch result {
-			case "Remove column":
```

Comment:
Why are the action types hardcoded?  I think they should be in a taxonomy.  Isn't part of this PR creating such a taxonomy as OPA data?